src/python/tests/core/platforms/android/adb_test.py

```diff
@@ -246,3 +248,15 @@ def test_apk_path_in_arg(self):
     """Test apk path passed as argument."""
     self.assertEqual(
         adb.get_package_name(self.test_apk_path), self.test_apk_pkg_name)
+
+
+@test_utils.slow
+class ResetUsbTest(android_helpers.AndroidTest):
+  """Tests for reset_usb."""
+
+  def setUp(self):
+    test_helpers.patch_environ(self)
```

Comment:
this is already done by android_helpers.AndroidTest, so setUp can be skipped.

Reply:
Done